example/credentials_azure_service_principal.tf

```diff
@@ -0,0 +1,14 @@
+resource "jenkins_folder" "azure_service_principal_test_folder" {
+    name        = "azure-service-principal-test-folder"
+    description = "A sample folder"
+}
```

Comment:
It doesn't appear as though this folder is being used. Is it needed?

Reply:
Thank you for the hint. I intentionally created the new folder as test folder for the `jenkins_credential_azure_service_principal` test.  I just updated the usage here. But if you want, I can use the example folder here. Can we resolve the conversation or should I update to the example folder and remove the new test folder?

Reply:
I think I like it going under the example folder -- that way it's a little more clear where all of the example credentials are being Terraformed into. Would you mind updating to use that and remove the `azure-service-principal-test-folder` folder?

Reply:
Sure. Updated it to use the example folder and removed the other folder.